script/build

```diff
@@ -5,18 +5,25 @@
 PLUGIN_NAME="${PLUGIN_NAME:?Must be set in .env}"
 PLUGIN_VERSION="${PLUGIN_VERSION:?Must be set in .env}"
 
+# This is necessary to get the project building on Go 1.6
+# Building the project will fail on Go 1.7+ as this env var will be ignored
+export GO15VENDOREXPERIMENT=0
```

Comment:
Yeah, that would be ideal. At the time of writing, I wasn't sure what would be required to support Go 1.7+. I raised an issue with CF CLI, and they're suggested the following patch to make it compile on all versions of Go:

main.go:170
```
-       // T needs to point to a translate func, otherwise cf internals blow up
-       i18n.T, _ = go_i18n.Tfunc("")
```

manifest_test.go:15
```
+type localeGetter struct{}
+
+func (l localeGetter) Locale() string {
+       return "en-us"
+}
+
 var _ = Describe("Manifest reader", func() {
        // testing code that calls into cf cli requires T to point to a translate func
-       i18n.T, _ = go_i18n.Tfunc("")
+       i18n.T = i18n.Init(localeGetter{})
```

If you think that would be a suitable patch, I will add that in as well.